### ISSUE TITLE
Feature Request: 

### ISSUE THREAD

**Issue by Medendorp** [grocy/grocy#2348]
Hi,

Is there a possibility to add loyalty cards to the shopping list? 

So to show a QR or a barcode when you are finished and you want to scan them so you can get your rewards?

Would be great!

gr Robert

**Comment by berrnd** [comment]
> Is there a possibility to add loyalty cards to the shopping list?

Beside Userfields, which are there to let you add any field you want to any entity (= a shopping list item, a product, everything) you want, there is no such possibility as of now.

**Comment by Medendorp** [comment]
Yes, I saw that also but I cant get that into the app I've installed on my phone.

**Comment by inson1** [comment]
I think Its good feature request, but i wouldnt add it to shipping list imho
